### PR TITLE
[CM H0] Fix grammar in export.md

### PR DESCRIPTION
**Pull request by Foberm**
## Basics

- [X] Short descriptions of your changes are in the release notes
      (added as entry in `doc/news/_preparation_next_release.md` which
      contains `_(my name)_`)
      **Please always add something to the release notes.**
- [X] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildservers are happy. If not, fix **in this order**:
  - [ ] add a line in `doc/news/_preparation_next_release.md`
  - [ ] reformat the code with `scripts/dev/reformat-all`
  - [ ] make all unit tests pass
  - [ ] fix all memleaks
- [ ] The PR is rebased with current master.

## Checklist

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I added code comments, logging, and assertions as appropriate (see [Coding Guidelines](https://master.libelektra.org/doc/CODING.md))
- [ ] I updated all meta data (e.g. README.md of plugins and [METADATA.ini](https://master.libelektra.org/doc/METADATA.ini))
- [ ] I mentioned [every code](/.reuse/dep5) not directly written by me in [reuse syntax](https://reuse.software/)

## Review

- [ ] Documentation is introductory, concise, good to read and describes everything what the PR does
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)
- [ ] APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)
- [ ] Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)

## Labels

- [ ] Add the "work in progress" label if you do not want the PR to be reviewed yet.
- [ ] Add the "ready to merge" label **if the basics are fulfilled** and no further pushes are planned by you.

### CODE REVIEW

File: doc/tutorials/export.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ for `/etc/hosts`.
 If no format is specified, the format `dump` will be used instead. The dump format is the standard way
 of expressing keys and all their relevant information. This format is intended to be used only within Elektra.
 The dump format is a good means of backing up Keys from the key database for use with Elektra later
-such as reimporting them later. As of this writing, `dump` is the only way to fully preserve all parts of the
+such as reimporting them. At the time of writing, `dump` is the only way to fully preserve all parts of the
 `KeySet`.
```
Comment (markus2330) [inline doc/tutorials/export.md:39]:
```suggestion
such as reimporting them.
```

**Review by markus2330 — COMMENTED** [review]
Thank you for creating the PR! I invited you! hugs

The sentence is actually outdated (there is also quickdump and mmapstorage implementing the full semantics), so let us better remove it.

**Comment by markus2330** [comment]
The commit messages should not contain `[CM H0]`, only the PR title. Please use `module: short sentence` as syntax for the first line of commit messages.

**Comment by vladfreeze** [comment]
Thank you for the invitation and the reply! I have already created the second PR on the private repository. How can module: short sentence be used as a syntax for the commit messages?

**Comment by markus2330** [comment]
> How can module: short sentence be used as a syntax for the commit messages?

You simply write that when git asks you for the commit message (usually pops up your favorite editor). In your case the module is simply `doc`, as you modify documentation.

**Comment by vladfreeze** [comment]
> > How can module: short sentence be used as a syntax for the commit messages?
> 
> You simply write that when git asks you for the commit message (usually pops up your favorite editor). In your case the module is simply `doc`, as you modify documentation.

Now I understand, will structure my commits like this in the future. Thank you!

**Pull request closed without merging**